src/main/java/uk/gov/justice/probation/courtcaseservice/service/DomainEventService.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.probation.courtcaseservice.service;
+
+import com.amazonaws.services.sns.model.MessageAttributeValue;
+import com.amazonaws.services.sns.model.PublishRequest;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.hmpps.sqs.HmppsQueueService;
+import uk.gov.justice.hmpps.sqs.HmppsTopic;
+import uk.gov.justice.probation.courtcaseservice.jpa.entity.DefendantEntity;
+import uk.gov.justice.probation.courtcaseservice.jpa.entity.HearingEntity;
+import uk.gov.justice.probation.courtcaseservice.service.model.event.DomainEventMessage;
+import uk.gov.justice.probation.courtcaseservice.service.model.event.DomainEventType;
+import uk.gov.justice.probation.courtcaseservice.service.model.event.PersonReference;
+import uk.gov.justice.probation.courtcaseservice.service.model.event.PersonReferenceType;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j
+@Service
+@AllArgsConstructor
+public class DomainEventService {
+
+    private final HmppsQueueService hmppsQueueService;
+    private final ObjectMapper objectMapper;
+    private final String EVENT_TYPE_KEY = "eventType";
+    @Value("${INGRES_URL}")
+    private String host;
+
+    private String HEARING_BY_HEARING_ID_TEMPLATE = "https://%s/hearing/%s";
+
+    private HmppsTopic getDomainEventTopic() {
+        return hmppsQueueService.findByTopicId("hmpps-domain-events");
+    }
+
+    void emitSentencedEvent(HearingEntity hearingEntity) {
+
+        var hmppsTopic = getDomainEventTopic();
+
+        var evtType = DomainEventType.SENTENCED_EVENT_TYPE;
```

Comment:
Does abbreviating `event` to `evt` achieve much here? For consistency with existing code (where we've tended to avoid them) I'd prefer if we avoided abbreviations unless there's a very good reason for them, i.e. if it's a very well known abbreviation relating to the business (like hmpps, cro, pnc etc) or if the name would otherwise be so long as to ruin readability. 

It makes it more easily searchable and saves a bit of cognitive load when things have one and only one spelling. We only write once and potentially read many times so we should optimise for readability.